src/components/Proposal/ActionButton.tsx

```diff
@@ -145,7 +155,7 @@ class ActionButton extends React.Component<IProps, IState> {
         { proposal.stage === IProposalStage.Queued && proposal.upstakeNeededToPreBoost.lte(new BN(0)) ?
             <button className={css.preboostButton} onClick={this.handleClickExecute.bind(this)} data-test-id="buttonBoost">
               <img src="/assets/images/Icon/boost.svg"/>
-              <span> Pre-Boost</span>
```

Comment:
these spaces are on purpose, did you check the design after removing them?

Reply:
no sorry about that, I'll put them back in